### DISCUSSION TITLE
2 questions (postgresql size & deleting files)

### DISCUSSION THREAD

**Discussion by abhay7dev** [forscht/ddrive#72] [Q&A] [answered]
As the title states, I have two quick questions.

On average, how big does the postgresql database get? How much space is required per database entry?

Also, in the 4.0 Update changelog, it is stated that
> deleting file is extremely faster because now ddrive don't have to delete files on discord channel and just need to remove from metadata only.

Does this mean that when a file is deleted, the chunks of the file are **not** deleted from the discord channel itself and they are deleted *only from the postgresql database*?

Thanks again :)

**Comment by forscht** [comment, marked as answer]
Sorry for late reply, somehow I am not getting notifications for discussion .
1. For 4 TB of data (each file is around 1GB) it took me around 30MB of postgres storage but it was before chunkSize was increased to 25MB now it should be around 20MB
2. Yes deleting file won't remove chunk from discord since we're using webhooks to upload file.

**Reply by abhay7dev** [reply]
Good to know. Thank you :)
